source/Classroom/Troubleshooting/Account_Administration/dont_know_your_username.md

```diff
@@ -15,8 +15,8 @@ Your SendGrid username can be any string of characters that works for you! If yo
 
 If you are unsure what your username is, consider checking your existing code or SMTP integration, as your username will probably be in there plain as day.
```

Comment:
Start by checking your existing code or SMTP integration to see if your username is plainly listed.